src/merchantPayment/hooks.ts

```diff
@@ -48,6 +48,7 @@ export function useMerchantPayments(apiBase: string, referenceId: string) {
       dispatch(showError(t('merchantWalletUnlockError')))
     }
 
+    // @ts-ignore
```

Comment:
will remove this when https://github.com/celo-org/payments/pull/35 is merged and we upgrade the payments sdk